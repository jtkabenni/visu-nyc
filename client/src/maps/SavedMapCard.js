import React from "react";

function SavedMapCard({ m }) {
  return (
    <div className="map-card">
      <img src={`${process.env.REACT_APP_DOMAIN}/${m.file}`}></img>
      <p>
        <b>
          {m.name} {m.user_username ? `by ${m.user_username}` : ""}
        </b>
      </p>

      <p>{m.note}</p>
    </div>
  );
}

export default SavedMapCard;
