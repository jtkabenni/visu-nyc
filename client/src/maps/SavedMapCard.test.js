import React from "react";
import { render } from "@testing-library/react";
import SavedMapCard from "./SavedMapCard";
let map = {
  name: "BK maps",
  use_username: "jennytak",
  file: "/",
  note: "bk map note",
};
test("renders SavedMapCard without crashing", () => {
  render(<SavedMapCard m={map} />);
});

it("matches snapshot", function () {
  const { asFragment } = render(<SavedMapCard m={map} />);
  expect(asFragment()).toMatchSnapshot();
});
