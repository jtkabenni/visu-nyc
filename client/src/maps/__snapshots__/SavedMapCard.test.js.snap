// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="map-card"
  >
    <img
      src="http://localhost:3001//"
    />
    <p>
      <b>
        BK maps 
      </b>
    </p>
    <p>
      bk map note
    </p>
  </div>
</DocumentFragment>
`;
